docs/architecture/test-strategy-and-standards-revision-3.md

```
@@ -118,6 +118,78 @@ dependencies {
+class MyIntegrationTest : FunSpec() {
+    @Autowired
+    private lateinit var mockMvc: MockMvc
+
+    init {
+        extension(SpringExtension())
+        test("my test") { /* dependencies available */ }
+    }
+}
+```
+
+**Anti-Pattern: Constructor Injection**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)